### PR TITLE
7 article votes

### PR DESCRIPTION
**Pull request by Farishta7**

### CODE REVIEW

**Comment by robert112233** [comment]
The functionality is there Farishta well done! Your code works fine however it is not optimistic. You should increase the votes by one onClick without waiting for a db response. The db response will update the page when you refresh. I think Dave might have covered this in the lecture this morning 👍

**Review by lewispricey — APPROVED** [review]
Thanks for implementing Rob's suggested changes! 

Good to merge when you're ready 🚀

**Pull request merged**
